src/Blade/ObjExt.cpp

```diff
@@ -1615,14 +1623,14 @@ B_BitMap *RM_GetResource(const char *name)
     resource = B_resource_manager.GetResource(2, name);
     if (resource != NULL)
     {
-        B_BitMap *bitMap = new B_BitMap((B_BitMap &)resource->data);
+        B_BitMap *bitMap = new B_BitMap((B_BitMap &)resource->file_name);
```

Comment:
What the reason of renaming `data` to `file_name`? It's not actually file name at this point.